demo_processes/pizza_order/src/main/groovy/org/camunda/bpm/pizza_order/PizzaOrderApplication.groovy

```diff
@@ -0,0 +1,30 @@
+package org.camunda.bpm.link3
+
+import org.camunda.bpm.application.ProcessApplication
+import org.camunda.bpm.application.impl.ServletProcessApplication
+import org.camunda.latera.bss.executionListeners.EventLogging
+import org.camunda.latera.bss.executionListeners.AutoSaveOrderData
+import org.camunda.latera.bss.logging.SimpleLogger
+import org.camunda.bpm.engine.delegate.ExecutionListener
+import org.camunda.bpm.engine.delegate.DelegateExecution
+
+@ProcessApplication("pizza_order")
+class PizzaOrderApplication extends ServletProcessApplication {
+  ExecutionListener getExecutionListener() {
+    return new ExecutionListener() {
+      void notify(DelegateExecution execution) {
+        new EventLogging().notify(execution)
+        if (execution.getEventName().equals(ExecutionListener.EVENTNAME_START) && (execution.getCurrentActivityName() != null )) {
```

Comment:
Okay. Save ```execution.getEventName()``` to a variable and use it twice. 

I know that Java-guys use ```equals```, but do you really need it for strings?